CHANGELOG.md

```diff
@@ -17,7 +16,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ### Removed
 
 ### Fixed
-
+- Projects overview not sorting on Likes High > Low [#557](https://github.com/DigitalExcellence/dex-frontend/issues/557)
```

Comment:
I don't like nitpicking, but I think it should be under changes and not fixes since the page was working fine :)

Reply:
Fair enough haha